pkg/controller/managedcluster/managedcluster_import_detach.go

```diff
@@ -39,13 +40,23 @@ func (r *ReconcileManagedCluster) importCluster(
 	//A clusterDeployment exist then get the client
 	if clusterDeployment != nil {
 		if !clusterDeployment.Spec.Installed {
-			return reconcile.Result{Requeue: true, RequeueAfter: 1 * time.Minute}, nil
+			return reconcile.Result{Requeue: true, RequeueAfter: 1 * time.Minute},
```

Comment:
error or info ? 😂  as cluster install can take 40+ minutes, we will expect a lot of error logs in that case?

Reply:
nvm, we also return error when failed to delete namespace 😂

Reply:
I just fixed it... testing...